src/main/java/com/hpe/adm/octane/ideplugins/services/mywork/DynamoMyWorkFilterCriteria.java

```diff
@@ -75,6 +75,10 @@
                         .and(createNativeStatusQuery("list_node.run_native_status.blocked", "list_node.run_native_status.not_completed", "list_node.run_native_status.planned")
                                 .and( Query.statement("parent_suite", QueryMethod.EqualTo, null)))
         );
+        filterCriteria.put(BDD_SCENARIO,
```

Comment:
Can we even test this anymore? This service is not the one that's being used currently, right?

Reply:
Nope, we cannot test this, my bad. It isn't the one that we use now